docs/website/root/manual/developer-docs/nodes/mithril-client-library.md

```diff
@@ -50,23 +50,31 @@ It handles the different types that can be queried to a mithril aggregator:
 
 :::caution
 
-Mithril Client library has not yet been published on [crates.io](https://crates.io/), so you won't be able to follow the procedure below. The crate will be published soon.
+Mithril client library has not yet been published on [crates.io](https://crates.io/), so you won't be able to follow the procedure below. The crate will be published soon.
+
+For now, you can experiment it by adding the dependency manually in the Cargo.toml of your project:
+
+```toml title="/Cargo.toml"
+mithril-client = { git = "https://github.com/input-output-hk/mithril.git" }
```

Comment:
Do we need to specify a path to the `mithril-client` crate?

Reply:
No, it just needs this line to be able to use the import of the library in a project.